src/patients/new/NewPatientForm.tsx

```diff
@@ -93,6 +98,7 @@ const NewPatientForm = (props: Props) => {
     <div>
       <form>
         <h3>{t('patient.basicInformation')}</h3>
+        {errorMessage && <Alert className="alert" color="danger" message={t(errorMessage)} />}
```

Comment:
No need for the call to `t` here. We can simply just pass error message to message since the error message was already internationalized.